### ISSUE TITLE
cataclysm experimental install fails

### ISSUE THREAD

**Issue by pnkhoody** [Homebrew/homebrew-games#449]
Regular install works fine: brew install homebrew/games/cataclysm --with-tiles

Experimental build install fails: brew install homebrew/games/cataclysm --with-tiles --HEAD

Error message in terminal: 

make[1]: *** [obj/map_iterator_test.o] Error 1
make[1]: *** [obj/creature_test.o] Error 1
1 error generated.
make[1]: *** [obj/npc_test.o] Error 1
1 error generated.
make[1]: *** [obj/monster_test.o] Error 1
1 error generated.
make[1]: *** [obj/line_test.o] Error 1
player_test.cpp:3:10: fatal error: 'morale.h' file not found
#include "morale.h"
         ^
1 error generated.
make[1]: *** [obj/player_test.o] Error 1
make: *** [tests] Error 2
make: *** Waiting for unfinished jobs....

**Comment by dunn** [comment]
Can you post `brew gist-logs cataclysm`, please? Thanks!

`HEAD` builds aren't always stable, so this might not be specific to Homebrew.  Can you try compiling outside of Homebrew and see if you get the same failure?

**Comment by mistydemeo** [comment]
I can repro building outside of Homebrew.

**Comment by mistydemeo** [comment]
Looks like this is a makefile bug. From the tests makefile:

```make
# Allow use of any header files from cataclysm.
# Catch sections throw unused variable warnings.
CXXFLAGS += -I../src -Wno-unused-variable
```

We're passing our own CXXFLAGS and our value isn't being appended to, so it's missing the include path it needs to build.

**Comment by pnkhoody** [comment]
Thanks mistydemeo. Guessing I don't need to include the logs now you've confirmed that? I can if needed.

**Comment by Anrock** [comment]
@mistydemeo so is there workaround or some kind of dirty hack to make it build? I've fiddled some time with CXXFLAGS but got nothing.

**Comment by Anrock** [comment]
Suddenly it builded fine yesterday.

**Comment by JosephCarrington** [comment]
I'm having a similar issue:

https://gist.github.com/8c069d13f9a84f65963e

**Comment by mistydemeo** [comment]
Pushed a fix for this and #482; `--HEAD` works for me now.
